src/content/pages/contacts.mdx

```diff
@@ -30,7 +30,7 @@ Follow us on these channels:
 - [EuroPython Telegram](https://t.me/+dEi3nIHzGRw0YjQy)
 - [EuroPython X Account](https://x.com/europython)
 
-Please use the hashtag [#EuroPython](https://x.com/hashtag/EuroPython)
+Please use the hashtag [#EuroPython](https://fosstodon.org/tags/EuroPython)
```

Comment:
Might it be better to remove the hyperlink to mastodon to make it clear it is not platform specific?

Reply:
Removed!